lib/jenkins_pipeline_builder/cli/pipeline.rb

```diff
@@ -31,20 +31,20 @@ def dump(job_name)
         Helper.setup(parent_options).dump(job_name)
       end
 
-      desc 'bootstrap Path', 'Generates pipeline from folder or a file'
+      desc 'bootstrap Path [ProjectName]', 'Generates pipeline from folder or a file'
       def bootstrap(path, project_name = nil)
         failed = Helper.setup(parent_options).bootstrap(path, project_name)
         exit(0) if failed.empty? # weird ordering, but rubocop decrees
         JenkinsPipelineBuilder.logger.error 'Encountered error during run'
         exit(1)
       end
 
-      desc 'pull_request Path', 'Generates jenkins jobs based on a git pull request.'
-      def pull_request(path, project_name = nil)
-        Helper.setup(parent_options).pull_request(path, project_name)
+      desc 'pull_request Path [ProjectName] [BaseBranchOnly]', 'Generates jenkins jobs based on a git pull request.'
+      def pull_request(path, project_name = nil, base_branch_only = false)
+        Helper.setup(parent_options).pull_request(path, project_name, base_branch_only)
```

Comment:
Hmmm, I think I should add some coercion to boolean here. I'll check to see if Thor already handles that.